CHANGELOG.md

```diff
@@ -53,7 +53,8 @@ of tasks being currently executed.
   - MailboxQuotaFixed matcher. Please use IsOverQuota instead.
 - UsersFileRepository, which was marked as deprecated for years
   - We accordingly removed deprecated methods within UsersRepositoryManagementMBean exposed over JMX (unsetAlias, getAlias, unsetForwardAddress, getForwardAddress). RecipientRewriteTables should be used instead.
-
+- JAMES-3016 RemoteDelivery now doesn't enable `allow8bitmime` property by default. More details about the property is at [java mail doc](https://javaee.github.io/javamail/docs/api/com/sun/mail/smtp/package-summary.html) 
```

Comment:
Add `This parameter could cause body content alteration leading to DKIM invalid DKIM signatures to be positionned. Thanks to Sergey B. for the report.`.